src/routes/lobby/components/LobbyPlayerIndicator.vue

```diff
@@ -149,9 +153,12 @@ export default {
   background-repeat: no-repeat;
   background-size: cover;
   position: relative;
-  animation: waving 5s;
-  animation-fill-mode: forwards;
   z-index: 1;
+  animation: waving 5s forwards;
+}
+
+.reverseWater {
```

Comment:
Does this work? I'm a little surprised that the specificity overrides the .water style block because they're both single class selectors. Maybe for explicitness' sake we should apply both here?
```suggestion
.water.reverseWater {
```

Reply:
I think is just preference, you might see it like, "this class belongs to the water component" and the suggestion works as well, but both options are doing the exact same thing. 

They are removing the existing animation `waving` for the new one `waving-reverse`. Personally I feel is more confusing to re-apply the `water` styles with `reverseWater`, because what matter to us is to apply the new animation.  But that's how I see it based on my personal interpretation.

Reply:
The thing that I don't like about the current way is that it is order-dependent. If `.water` style were moved below `.reverseWater` then `.water` would take precedence. At the cost of muddying the hierarchy, I'd rather ensure that the overriding style block has a higher-specificity selector to guarantee that it will apply.

Since it's a simple change, I'll commit it myself then merge

Reply:
I totally understand your point and I'll take that into consideration moving forward. Thanks for taking the time on doing the change